### PR TITLE
update changelog for ga

### PR DESCRIPTION
**Pull request by olipyskoty**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- N/A

### Description

This PR updates the `CHANGELOG.md` for GA 🥳

### Before review
* [X] Updated CHANGELOG.md if necessary
* [ ] Added unit tests if necessary
* [ ] Updated affected documentation
* [ ] Verified locally with `npm test`
* [ ] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [ ] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# 1.0.0 (December 17, 2021)
+
+The `1.0.0-beta.2` Twilio Video Room Monitor has been promoted to `1.0.0` and is now Generally Available!
```
Comment (timmydoza) [inline CHANGELOG.md:3]: Wondering about something like this instead:

```
The Twilio Video Room Monitor has been promoted to version `1.0.0` and is now Generally Available!
```

**Review by timmydoza — COMMENTED** [review]
Looks good! Just one thought.

**Review by sarahcstringer — APPROVED** [review]
LGTM

**Review by seancoleman2 — APPROVED** [review]
LGTM

**Review by timmydoza — APPROVED** [review]

**Pull request merged**